Assets/MRTK/Core/Services/BaseDataProviderAccessCoreSystem.cs

```diff
@@ -41,34 +41,36 @@ public override void Enable()
             }
         }
 
+        private static readonly ProfilerMarker UpdatePerfMarker = new ProfilerMarker("[MRTK] BaseDataProviderAccessCoreSystem.Update");
```

Comment:
It feels like just knowing that the base class is taking time isn't actually super helpful - if there were some way to also propagate the actual system that might make these markers more helpful here.

Reply:
i can look into this. in many cases, the derived class will also appear in the trace as they typically call the base as well as their own custom code.

Reply:
I think this should should be in the scope of the derived class' markers, this will show up in the profiler's hierarchy under that marker and we'll be able to see the split of time for each system

Reply:
@keveleigh, are you proposing adding implementations that are only markers and calls to the base class for each derived class? or are you indicating that the trace context should make it clear with the markers as they currently are?

Reply:
Per discussion, leaving this as-is and will evaluate if further refinements are needed after spending time using this change.